Source/GitSourceControl/Private/GitSourceControlUtils.cpp

```diff
@@ -587,6 +587,33 @@ bool GetRemoteBranchName(const FString& InPathToGitBinary, const FString& InRepo
 	return bResults;
 }
 
+// WCA EDIT - BEGIN
+bool GetRemoteBranchesWildcard(const FString& InPathToGitBinary, const FString& InRepositoryRoot, const FString& PatternMatch, TArray<FString>& OutBranchNames)
+{
+	TArray<FString> InfoMessages;
+	TArray<FString> ErrorMessages;
+	TArray<FString> Parameters;
+	Parameters.Add(TEXT("--remotes"));
+	Parameters.Add(TEXT("--list"));
+	bool bResults = RunCommand(TEXT("branch"), InPathToGitBinary, InRepositoryRoot, Parameters, { PatternMatch },
+								InfoMessages, ErrorMessages);
+	if (bResults && InfoMessages.Num() > 0)
```

Comment:
when using origin/* this includes ```origin/HEAD -> origin/main```  right? Does this break the feature, or does it not matter?

Reply:
Not at a computer but why would ‘git branch —list’ return origin/HEAD which is not a branch?

Reply:
Why, I don't know. but in all my git(lab) projects it does:

``` 
C:\Users\...>git branch --remote --list origin/*
  origin/HEAD -> origin/main
  origin/main
```

Reply:
Did you confirm this is broken by setting a breakpoint in the debugger?  

It seems atypical to define origin/* as a status branch pattern but if it breaks then we should handle it elegantly. 

These branch patterns were added to support common, standardized branching patterns in git like gitflow - https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

Reply:
![screenshot](https://i.imgur.com/B38rE2i.png)

See screenshot, confirmed that it adds it to the results

Reply:
Good find. Try `git branch --format='%(refname:short)' `